components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/util/AuthConfigurationUtil.java

```diff
@@ -131,6 +134,7 @@ public void buildResourceAccessControlData() {
      */
     public void buildClientAuthenticationHandlerControlData() {
 
+        SecretResolver secretResolver;
```

Comment:
why do we need this outside of the while loop?

Reply:
> why do we need this outside of the while loop?
fixed